python/tk_multi_workfiles/work_files.py

```diff
@@ -72,7 +73,7 @@ def __init__(self, app):
         self._work_area_template = None
         self._publish_template = None
         self._publish_area_template = None
-        
+
```

Comment:
SInce this is not our app, I would avoid doing not needed changes as this. Updating Tank apps is already painful sometimes, so less changes is the best. Thanks